src/debugulator_main.cpp

```diff
@@ -55,6 +57,11 @@ namespace
           return -1;
       }
 
+      if (vm.count(ebi::vcf::VERSION)) {
+          std::cout << "vcf-debugulator version " << VERSION_MAJOR << "." << VERSION_MINOR << std::endl;
```

Comment:
Could these messages be printed at the beginning of the help section too? Something like this:

```
vcf-debugulator version 0.6

Usage: vcf-debugulator [OPTIONS] [< input_file]
Allowed options:
  -h [ --help ]                 Display this help
  -v [ --version ]              Display version of the debugulator
  -i [ --input ] arg (=stdin)   Path to the input VCF file, or stdin
  -e [ --errors ] arg           Path to the errors report from the input VCF 
                                file
  -l [ --level ] arg (=warning) Validation level (error, warning, stop)
  -o [ --output ] arg (=stdout) Write to a file or stdout
```